MAUI/DataGrid/grid-events.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Grid Events in MAUI DataGrid control | Syncfusion
+description: Learn here all about Grid Events support in Syncfusion MAUI DataGrid (SfDataGrid) control and more about it.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Grid Events in MAUI DataGrid (SfDataGrid)
+
+## Cell Tap Events
+
+The `Datagrid` provides the following Events to handle interactions to the cells.
```

Comment:
The `DataGrid` provides the following events for handling interactions with the cells.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/bf715773-35c0-44e5-a23b-16432fbbbf5c)
Had updated as discussed.